q2_vsearch/tests/test_chimera.py

```diff
@@ -0,0 +1,206 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+
+import biom
+import numpy as np
+from qiime2.plugin.testing import TestPluginBase
+from qiime2.util import redirected_stdio
+from q2_types.feature_data import DNAFASTAFormat
+
+from q2_vsearch._chimera import (uchime_denovo, _uchime_denovo,
+                                 uchime_ref, _uchime_ref)
+from .test_cluster_features import _read_seqs
+
+
+class UchimeDenovoTests(TestPluginBase):
```

Comment:
In your tutorial there's a 3rd kind of category of sequence where it's "borderline chimeric" (and not listed in either file), are the conditions for that well understood enough to write a test for it?

Reply:
@ebolyen and I discussed offline and decided not to do this.